app/src/main/java/org/eyeseetea/malariacare/LoginActivity.java

```diff
@@ -134,11 +134,12 @@ private Intent propagateExtraAndResult(Intent intent){
         }
 
         if(getIntent().getBooleanExtra(SettingsActivity.SETTINGS_EULA_ACCEPTED, false)){
-            Log.i(TAG, "propagateExtraAndResult -> EULA accepted");
+            Log.i(TAG, "propagateExtraAndResult -> EULA accepted, Server overwrite from "+PreferencesState.getInstance().getDhisURL() +" to "+getUserIntroducedServer());
+            PreferencesState.getInstance().setDhisURL(getUserIntroducedServer());
```

Comment:
@idelcano the server introduced by the user in the login screen should not replace the server in the settings? Is this line changing this behaviour?